### PR TITLE
Typography - Homepage

### PR DESCRIPTION
**Pull request by dmarcoux**
Remove .text-muted.small classes from system status partial

Before:
system-status-before

After:
system-status-after

Underline time in status messages title 

Before:
status-message-before

After:
status-message-after

### CODE REVIEW

**Comment by dmarcoux** [comment]
I am thinking about limiting the text to 75 characters in the `System Status`, but it looks weird and I'm not sure how this should be fixed.

75ch

**Comment by hennevogel** [comment]
> not sure how this should be fixed

Untitled

**Comment by dmarcoux** [comment]
Good idea with the legend, I like it.

**Comment by hennevogel** [comment]
> I wonder though if some people refer to this.

I doubt it very much...

**Comment by dmarcoux** [comment]
The result:

system-status-legend

**Comment by dmarcoux** [comment]
[Homepage: Review font classes / tags and line length](https://trello.com/c/LgrseRWD/1208-homepage-review-font-classes-tags-and-line-length)

File: src/api/app/views/webui2/webui/main/_system_status_legend.html.haml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+%p
+  The above graphs show the number of active build jobs last week, currently
```
Comment (hennevogel) [inline src/api/app/views/webui2/webui/main/_system_status_legend.html.haml]: :thought_balloon:  It's not really above anymore :cowboy_hat_face:

**Comment by hennevogel** [comment]
@dmarcoux are you going to create a pattern? This definitely should be a thing in the app!

File: src/api/app/views/webui2/webui/main/_system_status_legend.html.haml (hunk 1 above)
Comment (dmarcoux) [reply src/api/app/views/webui2/webui/main/_system_status_legend.html.haml]: Ooops loll. I will change this.

**Comment by dmarcoux** [comment]
> @dmarcoux are you going to create a pattern? This definitely should be a thing in the app!

I will send a PR for this.

File: src/api/app/views/webui2/webui/main/_system_status_legend.html.haml (hunk 1 above)
Comment (dmarcoux) [reply src/api/app/views/webui2/webui/main/_system_status_legend.html.haml]: I improved the text a bit. It reads better now.

**Pull request merged**